conjure-java-jaxrs-client/src/main/java/com/palantir/conjure/java/client/jaxrs/JaxRsClient.java

```diff
@@ -55,4 +63,23 @@ private JaxRsClient() {}
                         config,
                         serviceConfiguration -> create(serviceClass, userAgent, hostEventsSink, serviceConfiguration)));
     }
+
+    /**
+     * Creates a {@code T client} for the given dialogue {@link Channel}.
+     */
+    public static <T> T create(Class<T> serviceClass, ConjureRuntime runtime, Channel channel) {
+        ObjectMapper objectMapper = ObjectMappers.newClientObjectMapper();
+        ObjectMapper cborObjectMapper = ObjectMappers.newCborClientObjectMapper();
```

Comment:
could these be static fields to allow sharing between clients?

Reply:
Might as well